### ISSUE TITLE
MFF2.py文件问题

### ISSUE THREAD

**Issue by MrYYYYYYYYYYY** [xsxjtu/MFF-SSIM#1]
作者您好
      这段时间在尝试复现您的实验时发现直接MS_LAP/test_cuda.ipynb这个demo无法复现您的结果，阅读代码后发现您在MFF2.py文件中259行处优化目标应该是写反了，修改为(1-ssim_out).backward()之后可以复现您的结果

**Comment by shuangxu96** [comment]
这是版本原因，请您使用kornia v0.1.3。

原因：
这是因为我代码中ssim的计算依赖于kornia。我的kornia版本是v0.1.3 （https://github.com/kornia/kornia/releases/tag/v0.1.2）。这个版本在输出ssim时，做了变换
loss = torch.clamp(1. - ssim_map, min=0, max=1) / 2.
此时，ssim和论文中的定义恰好相反。
在新的版本中，kornia定义了ssim和ssim_loss两个函数来区别是否做了这个变换，所以使用新版本的ssim进行优化无法复现ipynb中的结果。
